crates/pathfinder/src/storage/state.rs

```diff
@@ -1509,48 +1463,36 @@ mod tests {
 
             #[test]
             fn full() {
-                let storage = Storage::in_memory().unwrap();
-                let connection = storage.connection().unwrap();
+                with_default_blocks(|tx, _blocks| {
+                    StarknetBlocksTable::reorg(tx, StarknetBlockNumber::GENESIS).unwrap();
 
-                let blocks = create_blocks();
-                for block in &blocks {
-                    StarknetBlocksTable::insert(&connection, block).unwrap();
-                }
-
-                StarknetBlocksTable::reorg(&connection, StarknetBlockNumber::GENESIS).unwrap();
-
-                assert_eq!(
-                    StarknetBlocksTable::get(&connection, StarknetBlocksBlockId::Latest).unwrap(),
-                    None
-                );
+                    assert_eq!(
```

Comment:
Aren't we missing `StarknetBlocksTable::reorg(...)` here?

Reply:
Excellent find.

Reply:
Ah, it's there...